roles/cloud-ready/tasks/known_hosts.yml

```diff
@@ -0,0 +1,32 @@
+---
+- block:
+  - name: wait for new host to respond to ssh
+    wait_for:
+      port: 22
+      host: "{{ item.private_v4 }}"
+      search_regex: OpenSSH
+
+  - name: scan the ssh key of the new host via private address
+    command: "{{ 'ssh-keyscan -t rsa ' +  item.private_v4 }}"
+    register: keyscan_private_v4
+
+  - name: scan the ssh key of the new host via internal dns address
+    ignore_errors: yes
```

Comment:
Can fix after, but I prefer `failed_when: false` as it won't be red output, unlike `ignore_errors`.

Also, I prefer these controls to go after the module name, as a style thing. Again can be follow up.